src/components/Modal.js

```diff
@@ -173,158 +158,111 @@ const styles = {
       right: 10
     }
   },
-  header: {
+  titleBar: {
     backgroundColor: StyleConstants.Colors.PORCELAIN,
-    height: '50px',
-    left: 0,
-    position: 'absolute',
-    top: 0,
-    width: '100%'
-  },
-  headerText: {
+    padding: '15px 20px',
     color: StyleConstants.Colors.ASH,
-    fontSize: '13px',
-    left: '20px',
-    marginTop: '10px',
-    padding: '10px 0',
-    position: 'absolute',
+    fontSize: StyleConstants.FontSizes.SMALL,
     textTransform: 'uppercase'
   },
+  content: {
+    position: 'relative'
+  },
   footer: {
     backgroundColor: StyleConstants.Colors.PORCELAIN,
-    bottom: 0,
-    height: '50px',
-    left: 0,
-    position: 'absolute',
-    width: '100%'
+    padding: '15px 20px',
+    display: 'flex',
+    justifyContent: 'space-between'
```

Comment:
Love the use of flex here. I need to get in the habit of using flex more often. :+1: